### ISSUE TITLE
10. Update neighborhood exposure index

### ISSUE THREAD

**Issue by awunderground** [UI-Research/mobility-from-poverty#208]
Rename `share_black_nh_exposure`, `share_black_nh_exposure_quality`, `share_hispanic_exposure`, `share_hispanic_exposure_quality`, `share_other_nh_exposure`, `share_other_nh_exposure_quality`, `share_white_nh_exposure`, `share_white_nh_exposure_quality` **Note that the black_nh is incorrect. See note below. - Please disregard! I confused this with another metric

**Comment by cdsolari** [comment]
Please note that we incorrectly gave the wrong variable name just for blacks. In this case, Black includes Black Hispanics in this case. We have a table note about this too in our data tables. You coded this correctly, but the variable name is very misleading. Please change this to: share_black_exposure and share_black_exposure_quality. Apologies for the earlier mis-information.

**Comment by awunderground** [comment]
Are you sure? The code says "Black or African American alone" under "Not Hispanic or Latino".

```r
vars <- c(
  # Hispanic or Latino
  "DP05_0071E", # Estimate!!HISPANIC OR LATINO AND RACE!!Total population!!Hispanic or Latino (of any race)
  "DP05_0071M",
  # Not Hispanic or Latino
  "DP05_0077E", # White alone
  "DP05_0077M", # White alone MOE
  "DP05_0078E", # Black or African American alone
  "DP05_0078M", # Black or African American alone MOE
  "DP05_0079E", # American Indian and Alaska Native alone
  "DP05_0079M", # American Indian and Alaska Native alone MOE
  "DP05_0080E", # Asian alone
  "DP05_0080M", # Asian alone MOE
  "DP05_0081E", # Native Hawaiian and Other Pacific Islander alone
  "DP05_0081M", # Native Hawaiian and Other Pacific Islander alone MOE
  "DP05_0082E", # Some other race alone
  "DP05_0082M", # Some other race alone
  "DP05_0083E", # Two or more races
  "DP05_0083M"  # Two or more races MOE
)

```

I think the challenge with not being able to separate Black Hispanics is in the poverty exposure metric.

**Comment by cdsolari** [comment]
Darn, apologies. I thought this was poverty exposure. I see now that this one is intended to be racial diversity. The "neighborhood exposure" label in the issue threw me off. Please disregard...I will edit the Comment above.

**Comment by awunderground** [comment]
PR is #255

**Comment by awunderground** [comment]
Closed with #255
